src/04-modality-specific-files/02-magnetoencephalography.md

```diff
@@ -103,6 +135,17 @@ Whenever possible, please avoid using ad-hoc wording.
 
 Specific MEG fields MUST be present:
 
+{{ MACROS___make_metadata_table(
+   {
+      "SamplingFrequency": "REQUIRED",
```

Comment:
Here we should add something like: "The sampling frequency of data channels that deviate from the main sampling frequency SHOULD be specified in the `channels.tsv`file"

Reply:
```suggestion
      "SamplingFrequency": ("REQUIRED", "The sampling frequency of data channels that deviate from the main sampling frequency SHOULD be specified in the `channels.tsv` file."),
```